libs/blocks/merch-card-collection/merch-card-collection.js

```diff
@@ -291,8 +292,7 @@ export default async function init(el) {
   }
 
   const cardsRoot = await cardsRootPromise;
-  const overrides = el.dataset[OVERRIDE_PATHS];
-  const overridePromises = overrides?.split(',').map(fetchOverrideCard);
+  const overridePromises = mep?.custom?.[MERCH_CARD_COLLECTION_SELECTOR]?.map(fetchOverrideCard);
```

Comment:
look in personalization.js for the handleFragmentCommand function.  In particular, this line:
`if (config.mep.preview) a.dataset.manifestId = manifestPath;`

This is what makes the highlight feature possible.  I recommend doing this as well so people can see what has been changed.